src/controllers/dynakube/dynakube_controller.go

```diff
@@ -310,6 +312,18 @@ func (r *ReconcileDynaKube) reconcileActiveGateCapabilities(dkState *status.Dyna
 		}
 	}
 
+	//start automatic config creation
+	if dkState.Instance.Status.KubeSystemUUID != "" &&
+		dkState.Instance.Name != "" &&
```

Comment:
checking the `dkState.Instance.Name != ""` is not necessary, because the `DynaKube` must have a name to exist in Kubernetes